eth/downloader/peer.go

```diff
@@ -585,13 +589,26 @@ func (ps *peerSet) RTTs() []time.Duration {
 		rtts = append(rtts, p.rtt)
 		p.lock.RUnlock()
 	}
-	// Sort into ascending order and return them
+	// Sort into ascending order and retrieve the median
 	for i := 0; i < len(rtts); i++ {
 		for j := i + 1; j < len(rtts); j++ {
 			if rtts[i] > rtts[j] {
 				rtts[i], rtts[j] = rtts[j], rtts[i]
 			}
 		}
 	}
```

Comment:
If the slice is either []int or []float64 you can use sort.Ints or sort.Float64s